### PR TITLE
Add support for Django 4.0

### LINKED ISSUE #371: Officially support Django 4.0 on PyPI
PyPi only lists support of `django-environ` to Django <= 3.2 . I would like to see `django-environ` officially supporting Django 4.0 on PyPi.

### PR DESCRIPTION
**Pull request by browniebroke**
Fix #371

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -20,6 +20,7 @@ envlist =
     manifest
     py{35,36,37,38,39,310}-django{111,22}
     py{36,37,38,39,310}-django{30,31,32}
+    py{38,39,310}-django{40}
     pypy-django{111,22,30,31,32}
```
Comment (browniebroke) [inline tox.ini:24]: Django 4.0 requires Python 3.8+ so it doesn't install on PyPy

**Review by sergeyklay — APPROVED** [review]
Looks good to me. I'll merge a bit later.

**Review by jayvdb — APPROVED** [review]

**Comment by kennylajara** [comment]
Will you merge it?

**Review by sergeyklay — APPROVED** [review]

**Comment by sergeyklay** [comment]
Thank you for contributing and I'm sorry for the delay.

**Pull request merged**